### PR TITLE
Fix input to single-end MEGAHIT runs

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#716](https://github.com/nf-core/mag/pull/692) - Make short read processing a subworkflow (added by @muabnezor)
 - [#708](https://github.com/nf-core/mag/pull/708) - Fixed channel passed as GUNC input (added by @dialvarezs)
+- [#727](https://github.com/nf-core/mag/pull/727) - Fixed misspecified multi-FASTQ input for single-end data in MEGAHIT (reported by John Richards, fix by @jfy133)
```
Comment (jfy133) [inline CHANGELOG.md:20]:
```suggestion
- [#729](https://github.com/nf-core/mag/pull/729) - Fixed misspecified multi-FASTQ input for single-end data in MEGAHIT (reported by John Richards, fix by @jfy133)
```

**Comment by jfy133** [comment]
Closes https://github.com/nf-core/mag/issues/727

**Review by sateeshperi — APPROVED** [review]

**Review by sateeshperi — CHANGES_REQUESTED** [review]
fix tests

**Comment by jfy133** [comment]
Uh-oh....

**Comment by jfy133** [comment]
Replicated failure and this is the error:

```
(nf-core) james@bionb103:~/git/nf-core/mag/testing/work/17/c9f0a089e501713e8d3ba4fd0b5846 (fix-megahit)$ cat MEGAHIT-MetaBAT2-test_minigut.1.fa_busco.err
2024-12-11 17:04:42 ERROR:      list_of_reference_markers.bacteria_odb10.txt is not a valid option for 'placement_files'
2024-12-11 17:04:42 ERROR:      BUSCO analysis failed!
2024-12-11 17:04:42 ERROR:      Check the logs, read the user guide (https://busco.ezlab.org/busco_userguide.html), and check the BUSCO issue board on https://gitlab.com/ezlab/busco/issues

```

**Review by sateeshperi — APPROVED** [review]

**Pull request merged**